react/game/src/app/components/Navbar/components/Elements/index.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component, Fragment } from 'react';
+import { NavLink } from 'react-router-dom';
+
+import styles from '../../styles.scss';
+import LIST from '../../../../../constants/routes';
+import Logout from '../../../Logout';
+
+class Element extends Component {
+  menuList = obj => {
+    const mapped = [];
+    for (const item of Object.values(obj)) { // eslint-disable-line
+      if (item.name !== 'login') {
+        mapped.push(
+          <NavLink
+            exact
+            key={mapped.length}
```

Comment:
This is not fine. The key has to be different for each component. Here you always use the same key